packages/docs/src/styles/_examples.scss

```diff
@@ -213,6 +213,12 @@ $options-margin: $pt-grid-size * 3;
   animation: docs-wiggle-rotate $pt-transition-duration $pt-transition-ease infinite;
 }
 
+.docs-loading-skeleton-example-box {
+  flex: 1;
+  margin-right: $pt-grid-size * 1.5;
+  height: $pt-grid-size * 15;
```

Comment:
the bones vertically justify to the available space. if there is no inherent height in the containing parent, then the bones will be squashed together. of course we could add some vertical margins to make sure they are spaced at least X pixels from each other?